book/src/developers/contributing/releases/release_notes.md

```diff
@@ -0,0 +1,13 @@
+# Release notes
+
+**Prerequisite**: Release notes are generated with [towncrier](https://pypi.org/project/towncrier/). Ensure to have `towncrier` installed and the command is available.
```

Comment:
You'll need to update the [towncrier](https://towncrier.readthedocs.io/en/stable/tutorial.html) configration file [here](https://github.com/ethereum/trin/blob/master/pyproject.toml#L3) if we're moving the location of `release_notes.md`

Reply:
This section is just for developers to know how to generate the release notes. As in "here are important things to note when releasing".  I did not want to complicate the the book by integrating it with the generated release notes. We haven't released any notes yet either, so there is nothing to go in the book yet. 

I have left a note about where the release_notes are generated into. Let me know if you want to generate them into the book (I would lean towards having them in a release notes subheading in the `developers/progress_status.md` page)